Cython/Build/Dependencies.py

```diff
@@ -1040,6 +1044,16 @@ def copy_to_build_dir(filepath, root=os.getcwd()):
                     dir = os.path.dirname(c_file)
                     safe_makedirs_once(dir)
 
+                # write out the depfile, if requested
+                if depfile:
+                    src_base_dir, _ = os.path.split(source)
+                    relpaths = [os.path.relpath(fname, src_base_dir)
+                                for fname in deps.all_dependencies(source) ]
+                    depline = os.path.split(c_file)[1] + ": "
```

Comment:
Should we move all dependencies into new lines?
```suggestion
                    depline = os.path.split(c_file)[1] + ": \\\n  "
```